src/pages/overview/pdf-services-api/releasenotes.md

```diff
@@ -135,6 +135,11 @@ Upgrading to the latest SDK should not break existing applications.
 
 ## Change history
 
+### Extract PDF Server Side Release (16 May, 2022; server side release)
+| Change  | Language            | Description                                                                                   |
+| ------- | ------------------- | --------------------------------------------------------------------------------------------- |
+| Changed | All                 | Bug fixes and stability improvements.
```

Comment:
Missing | symbol at the end

Reply:
Is there any additional context we can provide here to our developers?